core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -49,18 +50,21 @@ import scala.util.control.NoStackTrace
   * Cluster mode can be set using the spark configuration
   * property spark.ext.h2o.mode which can be set in script starting sparkling-water or
   * can be set in H2O configuration class H2OConf
- */
+  */
 
 /**
   * Create new H2OContext based on provided H2O configuration
   *
   * @param sparkContext Spark Context
   * @param conf H2O configuration
   */
-class H2OContext private (@transient val sparkContext: SparkContext, @transient conf: H2OConf) extends org.apache.spark.Logging
+class H2OContext private (@transient val sparkContext: SparkContext, @transient conf: H2OConf, sqlc: SQLContext) extends org.apache.spark.Logging
```

Comment:
What about implicit parameter? We are already using that in `asDataFrame` calls.
Do we need SQLContext directly inside H2OContext?